triggers/service/auth/src/main/scala/com/daml/oauth/middleware/Server.scala

```diff
@@ -115,7 +111,9 @@ object Server extends StrictLogging {
             responseType = "code",
             clientId = config.clientId,
             redirectUri = toRedirectUri(request.uri),
-            scope = Some(login.claims.toQueryString),
+            // Auth0 will only issue a refresh token if the offline_access claim is present.
+            // TODO[AH] Make the request configurable, see https://github.com/digital-asset/daml/issues/7957
+            scope = Some("offline_access " + login.claims.toQueryString),
```

Comment:
Contrary to the other scopes this is actually standardized in oidc so a bit less important to make it configurable but still useful if we can do it in one go.